v21.2/deploy-lambda-function.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Create and Deploy an AWS Lambda Function on CockroachDB
+summary: Learn how to use AWS Lambda and CockroachDB Serverless.
+toc: true
+twitter: false
+referral_id: docs_lambda_python
+---
+
+This tutorial shows you how to create an [AWS Lambda](https://aws.amazon.com/lambda) function that communicates with a {{ site.data.products.serverless }} cluster.
+
+## Prerequisites
+
+Before starting the tutorial, do the following:
+
+1. Create a [{{ site.data.products.db }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account.
+
+1. Create an [AWS](https://aws.amazon.com/) account.
+
+1. Install the [AWS CLI](https://aws.amazon.com/cli/).
+
+1. Install [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+## Step 1. Create a {{ site.data.products.serverless }} cluster
+
+{% include cockroachcloud/quickstart/create-a-free-cluster.md %}
+
+## Step 2. Set up your cluster connection
+
+1. Once your cluster is created, the **Connection info** dialog displays. Follow the instructions to do the following:
+
+    1. Download the CockroackDB client.
+
+    1. Download the CA certificate.
+
+    1. Connect to the database with the [built-in SQL shell](cockroach-sql.html).
+
+1. In the SQL shell, create the `bank` database that your Lambda function will use:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    > CREATE DATABASE bank;
+    ~~~
+
+1. Exit the SQL shell:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    > \q
+    ~~~
+
+## Step 3. Get the sample code
+
+1. Clone the sample code's GitHub repo:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ git clone https://github.com/cockroachlabs/examples-aws-lambda/
+    ~~~
+
+1. In the `examples-aws-lambda/python` directory, create a new folder called `certs`.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ cd examples-aws-lambda/python
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ mkdir certs
```

Comment:
Can we just bundle the cert into the example app? @abhinavg6 I believe this is okay because the cert doesn't contain any private info.